runtime/doc/options.txt

```diff
@@ -2104,6 +2104,17 @@ A jump table for the options with a short description can be found at |Q_op|.
 	This option cannot be set from a |modeline| or in the |sandbox|, for
 	security reasons.
 
+						*'completefuzzycollect'* *'cfc'*
+'completefuzzycollect' 'cfc'	string	(default: empty)
+				global
+	option enables fuzzy collection in specific |ins-completion| modes,
+	adjusting how items are gathered for fuzzy matching based on input.
+	This option can contain the following values (separated by commas),
+	each enabling fuzzy collection for a specific completion mode:
+	k	keyword completion in 'complete' and current file
+	l	whole lines
+	f	file names
```

Comment:
can we use the values "keyword", "lines", "files"  instead? That way it is more intuitive.
Also I have to wonder, what about the other completion modes? Will that be done later? Or is not affected?

Reply:
> Also I have to wonder, what about the other completion modes? Will that be done later? Or is not affected?

not be affected. because they are all separated. If there are bugs, it will not affect daily use. If someone needs a certain mode to be included, then implement it here.

Reply:
> can we use the values "keyword", "lines", "files" instead? That way it is more intuitive.

okay

Reply:
Not affected means what? Other completion methods will be supporting "fuzzy collection" or not? Please clarify in the doc.

Reply:
I think the documentation has already explained that only keyword filename wholeline is supported. Those outside this range are not supported.